index.js

```diff
@@ -57,6 +57,8 @@ global.client = new Discord.Client({
 });
 global.bot = client;
 
+global.pollPingLastUsed = 0;
```

Comment:
nit: can we somehow stay away from using so much global variables?

Reply:
@domin-mnd This bot isn't going to be around for much longer, so I think I will just keep it as is for now.